pypeit/fluxcalibrate.py

```diff
@@ -29,14 +28,9 @@ class FluxCalibrate:
         outfiles (list, optional):
             Names of the output files.  If None, this is set to spec1dfiles and those are overwritten
     """
-    # Superclass factory method generates the subclass instance
-    @classmethod
-    def get_instance(cls, spec1dfiles, sensfiles, par=None, debug=False):
-        pypeline = fits.getheader(spec1dfiles[0])['PYPELINE'] + 'FC'
-        return next(c for c in cls.__subclasses__() if c.__name__ == pypeline)(
-            spec1dfiles, sensfiles, par=par, debug=debug)
+    def __init__(self, spec1dfiles, sensfiles, par=None, outfiles=None, debug=False):
 
-    def __init__(self, spec1dfiles, sensfiles, par=None, debug=False, outfiles=None):
+        from pypeit import sensfunc
```

Comment:
I did this to avoid a circulate import, since now fluxcalibrate and sensfunc depend on each other. If you can figure out a cleaner solution please implement it.

Reply:
I did this to avoid a circulate import but have now restructured things in a better way.